### PR TITLE
Update disclaimers in README and welcome page (closes #485)

### LINKED ISSUE #485: [0.5] update disclaimer in Kelp README
See doc - https://docs.google.com/document/d/1gk1huCpsc09R9G9h_gj_cODt6NBYS8Yu3wsmRyERfCc/edit

### PR DESCRIPTION
**Pull request by debnil**
This PR updates the disclaimers in the `README` and GUI welcome page, based on conversations with SDF Legal. It closes #485. There are no known limitations. 

Welcome Screen 1

Welcome Screen 2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,6 +24,12 @@ Kelp is built to:
 
 To learn more about the Stellar protocol check out [Stellar Videos on Coinbase Earn][stellar coinbase earn], or [this video about the Stellar DEX created by Lumenauts][sdex explainer video], or read more about it on the [Stellar Website][intro to stellar].
 
+## Be Smart and Go Slow
+
+**Important Disclaimer: Be Smart and Go Slow.** Whenever you trade on Stellar, you are trading with volatile assets, in volatile markets, and you risk losing money. Kelp is an experimental software that contains bugs. Use Kelp at your own risk. There is no guarantee you'll make a profit from using our bots or strategies. In fact, if you set bad parameters or market conditions change, Kelp might help you lose money very fast. So be smart and go slow.
+
+Your use of Kelp is governed by the Apache 2.0 open-source license. Please note that SDF’s interactions with you are governed by the SDF [Terms of Service](https://www.stellar.org/terms-of-service) and [Privacy Policy](https://www.stellar.org/privacy-policy).
```
Comment (nikhilsaraf) [inline README.md:31]: Can you move the two links at the bottom? only images have inline links
The links at the end of this file are ordered based on where they appear.

see **announcements distribution list** in the below paragraph as an example.

File: gui/web/src/components/molecules/Welcome/Welcome.js [hunk 2]
```diff
@@ -48,7 +48,7 @@ class Welcome extends Component {
 
     const kelpLogo = (
       <div className={styles.image}>
-        <img className={styles.symbol} src={symbol} alt="Kelp Symbol"/>
+        <img className={styles.symbol} src={symbol} alt="Kelp Symbol" />
```
Comment (nikhilsaraf) [inline gui/web/src/components/molecules/Welcome/Welcome.js:51]: can you configure your IDE to ignore whitespace changes for `.js` files?

The `.js` files are very inconsistent and PRs will be very noisy for both of us without this. It's fine for this commit.

File: gui/web/src/components/molecules/Welcome/Welcome.js [hunk 3]
```diff
@@ -61,15 +61,15 @@ class Welcome extends Component {
           </h3>
 
           <p className={styles.text}>
-          Kelp is a free and open-source trading bot for the Stellar Decentralized Exchange and centralized exchanges.
+            "Kelp is a free and open-source trading bot for the Stellar Decentralized Exchange and centralized exchanges. Kelp is programmed to support multiple trading schemes. This GUI of Kelp is limited to the Stellar Decentralized Exchange with only the buysell scheme.
           </p>
 
           <p className={styles.text}>
-          Kelp comes with multiple trading strategies out-of-the-box. This GUI of Kelp is limited to the Stellar Decentralized Exchange with the buysell strategy only.
+            You can use this GUI to define your own parameters to quickly get up and running with a trading bot in a matter of minutes.
```
Comment (nikhilsaraf) [inline gui/web/src/components/molecules/Welcome/Welcome.js:68]: `for each trading scheme` needs to be added between `parameters` and `to quickly get up and running...`

not sure if this modification was made in the doc before or after your PR

File: gui/web/src/components/molecules/Welcome/Welcome.js [hunk 4]
```diff
@@ -61,15 +61,15 @@ class Welcome extends Component {
           </h3>
 
           <p className={styles.text}>
-          Kelp is a free and open-source trading bot for the Stellar Decentralized Exchange and centralized exchanges.
+            "Kelp is a free and open-source trading bot for the Stellar Decentralized Exchange and centralized exchanges. Kelp is programmed to support multiple trading schemes. This GUI of Kelp is limited to the Stellar Decentralized Exchange with only the buysell scheme.
```
Comment (nikhilsaraf) [inline gui/web/src/components/molecules/Welcome/Welcome.js:64]: can you remove the `"` prefix?

**Review by nikhilsaraf — CHANGES_REQUESTED** [review]
some minor chanages.

can you also upload a screenshot of the GUI with the first two screens once updated?

File: gui/web/src/components/molecules/Welcome/Welcome.js (hunk 2 above)
Comment (debnil) [reply gui/web/src/components/molecules/Welcome/Welcome.js:51]: Got it. Will enable for subsequent PRs.

**Review by nikhilsaraf — APPROVED** [review]
LGTM, thanks!

**Pull request merged**